xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-api/src/main/java/org/xwiki/index/internal/TasksStore.java

```diff
@@ -48,6 +50,14 @@
 @Singleton
 public class TasksStore extends XWikiHibernateBaseStore
 {
+    /**
+     * A special character inserted at the beginning of all document versions, to make sure that the versions are never
+     * the empty string, otherwise Oracle fails to persist the task.
+     *
+     * @see <a href="https://jira.xwiki.org/browse/XWIKI-19571">XWIKI-19571</a></a>
+     */
+    private static final String FILLER = "-";
```

Comment:
Maybe add a note that it's temporary and will be removed as part of XWIKI-19581 (even if you have a TODO about that below in the code).